### PR TITLE
Add PullsURL to PushEventRepository

### PR DESCRIPTION
**Pull request by konojunya**
https://developer.github.com/v3/activity/events/types/#pushevent defines the statuses_url sent with each PushEvent webhook.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,5 +1,6 @@
 github.com/golang/protobuf v1.2.0 h1:P3YflyNX/ehuJFLhxviNdFxQPkGK5cDcApsge1SqnvM=
 github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
+github.com/google/go-github v17.0.0+incompatible h1:N0LgJ1j65A7kfXrZnUDaYCs/Sf4rEjNlfyDHW9dolSY=
```
Comment (gmlewis) [inline go.sum]: Can you please revert this file?  We don't want the dependency on the old version of this repo.

**Comment by gmlewis** [comment]
Also, please perform the `go generate ./...` step found in `CONTRIBUTING.md` to fix the Travis CI build failure.  Thanks.

File: go.sum (hunk 1 above)
Comment (konojunya) [reply go.sum]: ok

**Comment by konojunya** [comment]
I can rebase if you need :)

**Pull request merged**